src/block/expand/edit.js

```diff
@@ -47,7 +47,7 @@ const TEMPLATE = [
 		text: __( 'Show more', i18n ),
 		linkUrl: '#',
 		className: 'is-style-link stk-block-expand__show-button',
-		customAttributes: [ [ 'aria-hidden', 'false' ], [ 'role', 'button' ] ],
+		customAttributes: [ [ 'aria-hidden', 'false' ], [ 'role', 'button' ], [ 'aria-expanded', 'true' ] ],
```

Comment:
You are going to click this button to "show more", so this should be `aria-expanded` `false`
```suggestion
		customAttributes: [ [ 'aria-hidden', 'false' ], [ 'role', 'button' ], [ 'aria-expanded', 'false' ] ],
```